front/src/applications/stdcm/components/StdcmResults/StdcmResults.tsx

```diff
@@ -147,14 +188,18 @@ const StcdmResults = ({
             isReadOnly
             hideAttribution
             showStdcmAssets
+            isFeasible={!hasConflicts}
             setMapCanvas={setMapCanvas}
-            pathGeometry={selectedSimulation.outputs?.pathProperties.geometry}
-            simulationPathSteps={selectedSimulation.outputs?.results.simulationPathSteps}
+            pathGeometry={
+              hasConflicts
+                ? pathProperties?.geometry
+                : selectedSimulation?.outputs?.pathProperties?.geometry
+            }
+            simulationPathSteps={selectedSimulation?.outputs?.results?.simulationPathSteps}
           />
         </div>
       </div>
-      {/* TODO: Replace this part with the spaceTimeChartWithManchette component once it's merged (ui-manchette #504) */}
-      {isDebugMode && (
+      {isDebugMode && selectedSimulation?.outputs && !hasConflicts && (
```

Comment:
Should probably be enough

```suggestion
      {isDebugMode && selectedSimulation?.outputs && !hasConflicts && (
```